misk-aws-dynamodb-testing/src/main/kotlin/misk/aws/dynamodb/testing/InProcessDynamoDbModule.kt

```diff
@@ -21,9 +33,48 @@ class InProcessDynamoDbModule(
   constructor(vararg tables: KClass<*>) : this(tables.map { DynamoDbTable(it) })
 
   override fun configure() {
-    install(LocalDynamoDbModule(tables))
+    for (table in tables) {
+      multibind<DynamoDbTable>().toInstance(table)
+    }
     install(ServiceModule<InProcessDynamoDbService>())
     install(ServiceModule<CreateTablesService>()
         .dependsOn(InProcessDynamoDbService::class.toKey()))
   }
+
+  @Provides @Singleton
+  fun provideSqliteDbAccess(): SQLiteDBAccess {
+    Libsqlite4JavaLibraryPathInitializer.init()
+    return SQLiteDBAccess(null as File?)
+  }
+
+  @Provides @Singleton @Named("InProcessDynamoDb")
+  fun provideExecutorService(
+    executorServiceFactory: ExecutorServiceFactory
+  ): ExecutorService {
+    return executorServiceFactory.fixed("InProcessDynamoDb", 10)
```

Comment:
10 -> constant, maybe comment if it is significant or interesting to change?

Reply:
I used 10 threads ’cause it’s what DynamoDB uses as their default. It’s probably more than we need. I don’t want to put too much justification into that number because I think even they pulled it from thin air.